loops-and-arrays.js

```diff
@@ -44,18 +41,16 @@ for (let i = 0; i < numbers.length; i++) {
 }
 
```

Comment:
Hier geldt hetzelfde! dus ik zou in je for-loop/if-statement de array aanspreken en de waardes veranderen middels je if-statement, en daarna het resultaat loggen.
In dit geval:

for (let i = 0; i < numbers.length; i++) {
  if (numbers[i] % 2 === 0) {
    numbers[i] = numbers[i] * 2;
  }
  if (numbers[i] % 2 !== 0) {
    numbers[i] = numbers[i] * 3;
  }
  }

Reply:
En daarna de console.log natuurlijk